osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/GetConfigurationObjectServiceTest.java

```diff
@@ -62,36 +62,33 @@ public void setUp() {
     @Test
     public void getConfigurationObjectIOException() throws Exception {
 
-        // SETUP
-        when(this.dlmsConnection.get(any(AttributeAddress.class))).thenThrow(new IOException());
+            // SETUP
+            when(this.dlmsConnection.get(any(AttributeAddress.class))).thenThrow(new IOException());
 
-        assertThatExceptionOfType(ConnectionException.class).isThrownBy(() -> {
-            // CALL
+            assertThatExceptionOfType(ConnectionException.class).isThrownBy(() -> {// CALL
             this.instance.getConfigurationObject(this.conn);
         });
     }
 
     @Test
     public void getConfigurationObjectGetResultNull() throws Exception {
 
-        // SETUP
-        when(this.dlmsConnection.get(any(AttributeAddress.class))).thenReturn(null);
+            // SETUP
+            when(this.dlmsConnection.get(any(AttributeAddress.class))).thenReturn(null);
 
-        assertThatExceptionOfType(ProtocolAdapterException.class).isThrownBy(() -> {
-            // CALL
+            assertThatExceptionOfType(ProtocolAdapterException.class).isThrownBy(() -> {// CALL
             this.instance.getConfigurationObject(this.conn);
         });
     }
 
     @Test
     public void getConfigurationObjectGetResultUnsuccessful() throws Exception {
 
-        // SETUP
-        when(this.getResult.getResultCode()).thenReturn(AccessResultCode.READ_WRITE_DENIED);
-        when(this.dlmsConnection.get(any(AttributeAddress.class))).thenReturn(this.getResult);
+            // SETUP
+            when(this.getResult.getResultCode()).thenReturn(AccessResultCode.READ_WRITE_DENIED);
+            when(this.dlmsConnection.get(any(AttributeAddress.class))).thenReturn(this.getResult);
 
-        assertThatExceptionOfType(ProtocolAdapterException.class).isThrownBy(() -> {
-            // CALL
+            assertThatExceptionOfType(ProtocolAdapterException.class).isThrownBy(() -> {// CALL
```

Comment:
What is going on with these code formatting changes? Please restore the original formatting.

Reply:
No idea where this diff came from. I didn't touch this class as part of this story. Fixed anyway.